java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -232,6 +233,8 @@ public int getCommandedState() {
      * This method is not intended for general use, e.g. for users to set the
      * KnownState, so it doesn't appear in the Turnout interface.
      *
+     * On change, fires Property Change "KnownState".
```

Comment:
Sorry that I missed this before, but if you want this comment on a separate line, you need to add `<p>` before this line since it is converted to html when Javadoc is parsed. See line 218 and line 232 there you have `<p>`.

But you don't need `<p>` after this line since Javadoc will always place `@param` on its own line.